patches/0001_js_mse_eme_mvt.patch

```diff
@@ -402,28 +404,35 @@ index c040757..bf5192a 100644
      document.body.appendChild(createElement('div', 'testview'));
  
      var div = document.getElementById(this.divId);
-@@ -150,12 +151,17 @@ function TestView(testSuiteVer) {
+@@ -149,14 +150,21 @@ function TestView(testSuiteVer) {
+       switchDiv.lastChild.exec = switchDiv.lastChild.onclick;
      }
      for (var i = 0; i < selectors.length; ++i) {
-       switchDiv.appendChild(document.createTextNode(selectors[i].text));
+-      switchDiv.appendChild(document.createTextNode(selectors[i].text));
 -      var select = document.createElement('select');
-+      var select = document.createElement('span');
-+      select.id = selectors[i].id;
-       for (var j = 0; j < selectors[i].optionTexts.length; ++j) {
+-      for (var j = 0; j < selectors[i].optionTexts.length; ++j) {
 -        select.appendChild(createOption(selectors[i].optionTexts[j],
 -            selectors[i].values[j]));
-+        select.appendChild(createAnchor(selectors[i].optionTexts[j],selectors[i].optionTexts[j]));
-+        select.lastChild.setAttribute('data-href', location.origin + "?test_type=" + harnessConfig.testType + "&engine_" + selectors[i].id + "=" + selectors[i].optionTexts[j]);
-+        select.lastChild.onclick = window.navigate;
-+        if (selectors[i].optionTexts[j] != selectors[i].defaultValue)
-+            select.lastChild.classList.add('focusable');
-+        else
-+            select.lastChild.classList.add('bold');
++      if (harnessConfig.testType.includes(selectors[i].id)) {
++        switchDiv.appendChild(document.createTextNode(selectors[i].text));
++        var select = document.createElement('span');
++        select.id = selectors[i].id;
++        for (var j = 0; j < selectors[i].optionTexts.length; ++j) {
++          select.appendChild(createAnchor(selectors[i].optionTexts[j],selectors[i].optionTexts[j]));
++          select.lastChild.setAttribute('data-href', location.origin + "?test_type=" + harnessConfig.testType + "&engine_" + selectors[i].id + "=" + selectors[i].optionTexts[j]);
```

Comment:
This line is far too long, please split it somehow. You can use a temporary variable for the url:
```
          let url = location.origin;+ "?test_type=" + harnessConfig.testType;
          url += "&engine_" + selectors[i].id + "=" + selectors[i].optionTexts[j];
          select.lastChild.setAttribute('data-href', url);
```